datasets/sick/README.md

```diff
@@ -0,0 +1,182 @@
+    "entailment_AB": "A_neutral_B",
+    "entailment_BA": "B_neutral_A",
+    "label": 1,
+    "id": "1",
+    "relatedness_score": 4.5,
+    "sentence_A": "A group of kids is playing in a yard and an old man is standing in the background",
+    "sentence_A_dataset": "FLICKR",
+    "sentence_A_original": "A group of children playing in a yard, a man in the background.",
+    "sentence_B": "A group of boys in a yard is playing and a man is standing in the background",
+    "sentence_B_dataset": "FLICKR",
+    "sentence_B_original": "A group of children playing in a yard, a man in the background."
+}
+```
+
+### Data Fields
+
+- pair_ID: sentence pair ID
+- sentence_A: sentence A
+- sentence_B: sentence B
+- label: textual entailment gold label (NEUTRAL, ENTAILMENT, or CONTRADICTION)
+- relatedness_score: semantic relatedness gold score (on a 1-5 continuous scale)
+- entailment_AB: entailment for the A-B order (A_neutral_B, A_entails_B, or A_contradicts_B)
+- entailment_BA: entailment for the B-A order (B_neutral_A, B_entails_A, or B_contradicts_A)
+- sentence_A_original: original sentence from which sentence A is derived
+- sentence_B_original: original sentence from which sentence B is derived
+- sentence_A_dataset: dataset from which the original sentence A was extracted (FLICKR vs. SEMEVAL)
+- sentence_B_dataset: dataset from which the original sentence B was extracted (FLICKR vs. SEMEVAL)
+
+### Data Splits
+
+Train Trial Test
+4439 495 4906
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+[Needs More Information]
+
+### Licensing Information
+
+[Needs More Information]
+
+### Citation Information
+
+```
+@inproceedings{marelli-etal-2014-sick,
+    title = "A {SICK} cure for the evaluation of compositional distributional semantic models",
+    author = "Marelli, Marco  and
+      Menini, Stefano  and
+      Baroni, Marco  and
+      Bentivogli, Luisa  and
+      Bernardi, Raffaella  and
+      Zamparelli, Roberto",
+    booktitle = "Proceedings of the Ninth International Conference on Language Resources and Evaluation ({LREC}'14)",
+    month = may,
+    year = "2014",
+    address = "Reykjavik, Iceland",
+    publisher = "European Language Resources Association (ELRA)",
+    url = "http://www.lrec-conf.org/proceedings/lrec2014/pdf/363_Paper.pdf",
+    pages = "216--223",
+}
+```
```

Comment:
````suggestion
```

### Contributions

Thanks to [@calpt](https://github.com/calpt) for adding this dataset.
````
we have a new section :)